src/main/java/uk/gov/hmcts/juror/api/moj/repository/letter/court/ExcusalRefusalLetterListRepositoryImpl.java

```diff
@@ -1,79 +1,193 @@
+            .where(QJurorPool.jurorPool.isActive.isTrue()
+                .and(QJuror.juror.excusalRejected.eq("Y"))
+                .and(QJurorPool.jurorPool.owner.eq(owner))
+                .and(QPoolRequest.poolRequest.poolNumber.in(poolNumbers)))
+            .fetch();
+    }
+
+    private void filterEligibleLetterSearchCriteria(JPAQuery<Tuple> jpaQuery,
                                                     CourtLetterSearchCriteria courtLetterSearchCriteria) {
         if (!StringUtils.isEmpty(courtLetterSearchCriteria.jurorNumber())) {
-            jpaQuery.where(EXCUSAL_REFUSED_LETTER_LIST.jurorNumber.startsWith(courtLetterSearchCriteria.jurorNumber()));
+            jpaQuery.where(QJurorPool.jurorPool.juror.jurorNumber.startsWith(courtLetterSearchCriteria.jurorNumber()));
         }
 
         if (!StringUtils.isEmpty(courtLetterSearchCriteria.jurorName())) {
-            jpaQuery.where(EXCUSAL_REFUSED_LETTER_LIST.firstName.concat(" " + EXCUSAL_REFUSED_LETTER_LIST.lastName)
+            jpaQuery.where(QJurorPool.jurorPool.juror.firstName.concat(" ").concat(QJurorPool.jurorPool.juror.lastName)
                 .containsIgnoreCase(courtLetterSearchCriteria.jurorName()));
         }
 
         if (!StringUtils.isEmpty(courtLetterSearchCriteria.postcode())) {
             jpaQuery.where(
-                EXCUSAL_REFUSED_LETTER_LIST.postcode.trim().equalsIgnoreCase(courtLetterSearchCriteria.postcode()
-                    .trim()));
+                QJurorPool.jurorPool.juror.postcode.trim()
+                    .eq(courtLetterSearchCriteria.postcode().trim().toUpperCase()));
         }
 
         if (!StringUtils.isEmpty(courtLetterSearchCriteria.poolNumber())) {
-            jpaQuery.where(EXCUSAL_REFUSED_LETTER_LIST.poolNumber.startsWith(courtLetterSearchCriteria.poolNumber()));
-        }
-
-        if (!courtLetterSearchCriteria.includePrinted()) {
-            jpaQuery.where(EXCUSAL_REFUSED_LETTER_LIST.datePrinted.isNull());
+            jpaQuery.where(QJurorPool.jurorPool.pool.poolNumber.startsWith(courtLetterSearchCriteria.poolNumber()));
         }
     }
 
-    private void orderExcusalRefusedQueryResults(JPAQuery<ExcusalRefusedLetterList> jpaQuery,
-                                                 boolean isIncludePrinted) {
+    private void orderExcusalRefusedQueryResults(JPAQuery<Tuple> jpaQuery) {
 
-        if (isIncludePrinted) {
-            jpaQuery.orderBy(new CaseBuilder()
-                .when(EXCUSAL_REFUSED_LETTER_LIST.datePrinted.isNull())
-                .then(0)
-                .otherwise(1)
-                .asc());
-        }
-        jpaQuery.orderBy(EXCUSAL_REFUSED_LETTER_LIST.datePrinted.desc());
-        jpaQuery.orderBy(EXCUSAL_REFUSED_LETTER_LIST.jurorNumber.asc());
+        jpaQuery.orderBy(QJurorHistory.jurorHistory.dateCreatedDateOnly.desc(),
+            QJurorPool.jurorPool.juror.jurorNumber.asc());
```

Comment:
As you join on QJuror.juror we should filter on QJuror.juror to prevent duplicate joins